README.md

```diff
@@ -144,6 +144,60 @@ Meanwhile, the pipeline of Project A was hanging in the **Can I Deploy?** until
 **Disclaimer**: You can see these building blocks in our github actions build file, but the flow there looks a little bit different whereas the support pipelines from the providers are simulated.
 
 
+# Setting up the Honeycomb Starter Pack
+
+The `/terraform` folder contains the config to set up a starter pack for exploring build events in Honeycomb. This includes setting up a derived column, queries and a board 
```

Comment:
This is probably me nitpicking/lacking of knowledge - not 100% clear which `/terraform` folder you were referring to exactly here?

Reply:
fair enough 👍 - I can add a link to it also I guess it should be terraform/ not the other way too